### PR TITLE
Add minor changes

### PR DESCRIPTION
**Pull request by kairusds**
~~TODO: Need someone to test 5d3e754 with a disposable sdcard~~

### CODE REVIEW

**Comment by kairusds** [comment]
TODO: Fix exit dialog not appearing

File: src/ru/nsu/ccfit/zuev/osu/MainScene.java [hunk 1]
```diff
@@ -1061,6 +1064,27 @@ public void run() {
...
+                    public void onClick(DialogInterface dialog, int which) {
+                        exit();
+                        PowerManager.WakeLock wakeLock = GlobalManager.getInstance().getMainActivity().getWakeLock();
+                        if (wakeLock != null && wakeLock.isHeld()) {
+                            wakeLock.release();
+                        }
+                    }
+                });
+                builder.setNegativeButton("No", null);
+                builder.show();
```
Comment (EdrowsLuo) [inline src/ru/nsu/ccfit/zuev/osu/MainScene.java:1083]: Do not use hardcode strings, you should use resource id instead.

Comment (kairusds) [reply src/ru/nsu/ccfit/zuev/osu/MainScene.java:1083]: okay, will do right ahead

**Comment by EdrowsLuo** [comment]
> somehow this failed to compile on the github's ci but it worked fine when i did it on my machine with sdk 27 and 29 both installed

\* What went wrong:
Execution failed for task ':lint'.
\> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  
  Errors found:
  
  /home/runner/work/osu-droid/osu-droid/src/ru/nsu/ccfit/zuev/audio/serviceAudio/SongService.java:359: Error: Field requires API level 29 (current min is 14): android.app.TaskInfo#topActivity [NewApi]
          ComponentName componentName = activityManager.getRunningTasks(1).get(0).topActivity;
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

ci has told you the problem, you should wrap these codes with a system api level check.
Because if you do not do this, low api level system will crash when run these codes.

**Pull request merged**